src/styles/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  &__nav {
+    &__logo-img {
+      display: flex;
```

Comment:
The `display: flex;` property seems unnecessary here. The `display` property affects the layout of the children of the element, but since `.header__nav__logo-img` doesn't have any child elements, it doesn't do anything. You should remove it.